Gopkg.toml

```diff
@@ -5,3 +5,7 @@
 [[constraint]]
   name = "github.com/gobuffalo/packr"
   version = "1.12.0"
+
+[[constraint]]
+  name = "github.com/gferraro/go-api"
```

Comment:
I think you mean the TheCacophonyProject repo, not this one. Get any go-api changes landed before merging this.

Reply:
yep just there for me to test with at the moment